iina/InitialWindowController.swift

```diff
@@ -224,20 +170,14 @@ class InitialWindowController: NSWindowController {
 
   private func setMaterial(_ theme: Preference.Theme?) {
     guard let window = window, let theme = theme else { return }
-    if #available(macOS 10.14, *) {
-      window.appearance = NSAppearance(iinaTheme: theme)
-      if #available(macOS 10.16, *) {
-        let gradientLayer = CAGradientLayer()
-        gradientLayer.colors = window.effectiveAppearance.isDark ?
-          [NSColor.black.withAlphaComponent(0.4).cgColor, NSColor.black.withAlphaComponent(0).cgColor] :
-          [NSColor.black.withAlphaComponent(0.1).cgColor, NSColor.black.withAlphaComponent(0).cgColor]
-        leftOverlayView.wantsLayer = true
-        leftOverlayView.layer = gradientLayer
-      }
-    } else {
-      window.appearance = NSAppearance(named: .vibrantDark)
-      mainView.layer?.backgroundColor = CGColor(gray: 0.1, alpha: 1)
-      visualEffectView.material = .ultraDark
+    window.appearance = NSAppearance(iinaTheme: theme)
+    if #available(macOS 10.16, *) {
```

Comment:
I recall something about needing to retain #available referencing 10.16 instead of 11 due to the need to support old SDKs. We might want to consider cutting off support for older Xcode versions, updating this, and allow use of the latest Swift features. But first we should merge a lot of the outstanding PRs before doing refactoring that can introduce merge conflicts.